src/ServiceControl.Monitoring/Infrastructure/EndpointInstanceActivityTracker.cs

```diff
@@ -17,14 +17,13 @@ public void Record(EndpointInstanceId instanceId, DateTime utcNow)
 
         public bool IsStale(EndpointInstanceId endpointInstance)
         {
-            DateTime lastActivityTime;
-            if (endpointsInstances.TryGetValue(endpointInstance, out lastActivityTime))
+            if (endpointsInstances.TryGetValue(endpointInstance, out var lastActivityTime))
             {
-                var timeSpan = DateTime.UtcNow - lastActivityTime;
-                return timeSpan > StalenessThreshold;
+                var age = DateTime.UtcNow - lastActivityTime;
+                return age > StalenessThreshold;
             }
 
-            return false;
+            return true;
```

Comment:
This is a change in behaviour and causes `When_no_endpoint_instance_reports_not_stale` to fail. Is this change intentional?